pixmappy/PixelMapCollection.py

```diff
@@ -9,8 +9,8 @@
 
 All transformations can be done on arrays of coordinates as well as scalars.
 
-Pixel or world positions are to be numpy arrays of shape (2) or (N,2).  Sky
-positions are astropy.coordinates.SkyCoord objects/arrays.
+Pixel or world positions are to be numpy arrays of shape (2) or (N,2).
+Sky positions are RA, Dec in degrees.
```

Comment:
Comment needs change since positions are now input as either 2 scalars or 2 arrays of matching length N.  You can show me this week how I would just change this myself instead of making a comment about it!